src/core/public/overlays/flyout.tsx

```diff
@@ -19,8 +19,9 @@
 
 import React from 'react';
 
+import { Subject } from 'rxjs';
+
 import { EuiFlyout } from '@elastic/eui';
-import { EventEmitter } from 'events';
 import ReactDOM from 'react-dom';
 import { I18nContext } from 'ui/i18n';
```

Comment:
@epixa I assume that's just wrong doing this from a core service. Could you explain me what's the correct way to import that `I18nContext` is now?

Reply:
`i18n` passed as a core dependency in `start` lifecycle method. you can find example in other core services https://github.com/elastic/kibana/blob/master/src/core/public/legacy/legacy_service.ts#L57-L72